### ISSUE TITLE
[FUNCTIONALITY]: Process List - SHOULD have a React component

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9198]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our [process list component](https://design.va.gov/components/process-list) should have a design system component so we can make changes easier and ensure 1:1 with design styles.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Component matches existing HTML structure and styling
- [ ] Component includes unit tests with axe check

## Environment

* https://design.va.gov/components/process-list

## Screenshots or Trace Logs

<img width="890" alt="Screen Shot 2020-05-18 at 10 47 55 AM" src="https://user-images.githubusercontent.com/934879/82233078-0f572400-98f5-11ea-9b23-b455e565dce3.png">

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #228](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/228) via [**ZenHub**](https://www.zenhub.com/)
